### PR TITLE
Add Google Cloud SQL config for PG CDC

### LINKED ISSUE #2362: Document steps for configuring Google Cloud SQL CDC
https://risingwave-community.slack.com/archives/C03BW71523T/p1720604180563529

https://docs.risingwave.com/docs/current/ingest-from-mysql-cdc/
https://docs.risingwave.com/docs/current/ingest-from-postgres-cdc/

### PR DESCRIPTION
**Pull request by WanYixian**
## Description

Add how to enable logical replication on Google Cloud SQL for PG and MySQL CDC

## Related code PR

https://github.com/risingwavelabs/risingwave/issues/17729
## Related doc issue

Resolve #2362 

## Checklist

- [ ] I have checked the doc site preview, and the updated parts look good.
- [ ] I have acquired the approval from the owner (and optionally the reviewers) of the code PR and at least one tech writer (`emile-00`, `hengm3467`, & `WanYixian`).

### CODE REVIEW

File: docs/guides/ingest-from-mysql-cdc.md [hunk 1]
```diff
@@ -119,6 +119,23 @@ The configuration process is different for AWS RDS MySQL or Aurora (MySQL-Compat
 
 6. Ensure your MySQL users can access the tables and replications.
 
+</TabItem>
+<TabItem value="Google Cloud SQL" label="Google Cloud SQL">
```
Comment (StrikeW) [inline docs/guides/ingest-from-mysql-cdc.md]: cloud sql is not a single product, it includes MySQL and Postgres. The steps to enable `logical replication` and alter role is only applicable to PG not MySQL.

Comment (WanYixian) [reply docs/guides/ingest-from-mysql-cdc.md]: Removed, please take another look

File: docs/guides/ingest-from-postgres-cdc.md [hunk 2]
```diff
@@ -133,6 +133,23 @@ Here we will use a standard class AWS RDS PostgreSQL instance without Multi-AZ d
...
 
+</TabItem>
+<TabItem value="Google Cloud SQL" label="Google Cloud SQL">
+
+To enable logical replication on Google Cloud SQL, follow these steps:
+
+1. Go to Google Cloud Console, locate your Cloud SQL instance and set `cloudsql.logical_decoding` parameter to `On`:
+  ![Set parameter to on](../images/google-cloud-sql-1.png)
+
+2. Go to **Connections** > **Networking**, authorize the service to connect to Cloud SQL using the network CIDR：
```
Comment (StrikeW) [inline docs/guides/ingest-from-postgres-cdc.md]: nits: `authorize the RisingWave to connect to ...`

**Review by StrikeW — APPROVED** [review]
Rest LGTM

**Pull request merged**